src/components/StatsPanel/StatsPanel.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { styled } from '@mui/material/styles';
+
+import type { PaletteColor } from '@mui/material/styles';
+
+export const CardContainer = styled('div')(({ theme }) => ({
+  backgroundColor: theme.palette.background.paper,
+  borderRadius: theme.spacing(2.5),
+  padding: theme.spacing(4),
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(3),
+  width: theme.spacing(32.5),
```

Comment:
CardContainer sets a fixed width using theme.spacing(32.5). This is a very large fixed width expressed via spacing and may cause layout inflexibility and poor responsiveness. Using fractional spacing values also produces fractional pixel values which can cause blurry results in some browsers.

**Sugerencia:**
```typescript
Prefer responsive widths (maxWidth) or use a smaller integer spacing based on the design system. Example:

width: '100%',
maxWidth: theme.spacing(40),

Or choose an integer theme.spacing value: theme.spacing(32) instead of 32.5 and prefer responsive breakpoints if needed.
```